ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandStore.java

```diff
@@ -95,6 +95,10 @@ public static CommandStore create(
           ConsumerConfig.ISOLATION_LEVEL_CONFIG,
           IsolationLevel.READ_COMMITTED.toString().toLowerCase(Locale.ROOT)
       );
+      kafkaConsumerProperties.put(
+          ConsumerConfig.AUTO_OFFSET_RESET_CONFIG,
+          "none"
```

Comment:
is there a constant somewhere for this?

Reply:
I don't really think so. https://kafka.apache.org/22/javadoc/index.html?org/apache/kafka/clients/consumer/ConsumerConfig.html

The closest thing is `StreamsConfig. NO_OPTIMIZATION` which is `none` but that doesn't really make sense here.